sle12/templates/csv/mount_options.csv

```diff
@@ -6,14 +6,4 @@
 #     '$' to reference a variable, e.g. var_removable_partition,nodev)
 #  If the remediation can create (i.e. not just modify) an /etc/fstab line,
 #  add the 'create_fstab_entry_if_needed' literal string as the third argument.
-/dev/shm,nodev
```

Comment:
It is not obvious to me why these are going away in SLE12. Perhaps could split the commit or add a comment on why these are getting removed?

Reply:
I am 99% sure this was an accident, I have updated the PR.